File: Project/Backend/daariiou_photography_backend/daariiou_photography_backend/DTO/ShootingDTO.cs
using daariiou_photography_backend.Models;
using System;
using System.Collections.Generic;

#nullable disable

namespace daariiou_photography_backend.DTO
{
    public class ShootingDTO
    {
        public int Sid { get; set; }
        public DateTime Date { get; set; }
        public string Remarks { get; set; }
        public string ReasonDeclined { get; set; }
        public string Status { get; set; }
        public int Uid { get; set; }
        public int KosId { get; set; }

        public Models.KindOfShooting KindOfShooting { get; set; }
        public User User { get; set; }
    }
}

File: Project/Frontend/daariiou-photography-frontend/src/api/lib/models/shooting-dto.ts
/* tslint:disable */
/* eslint-disable */
import { KindOfShooting } from './kind-of-shooting';
import { User } from './user';
export interface ShootingDto {
  date?: string;
  kindOfShooting?: null | KindOfShooting;
  kosId?: number;
  reasonDeclined?: null | string;
  remarks?: null | string;
  sid?: number;
  status?: null | string;
  uid?: number;
  user?: null | User;
}
